File: ch4 supplemental/repl6.cs
using System;
class MainClass {
	public static void Main (string[] args) {
		Console.WriteLine("Enter a positive number");
		
		int positiveNumberAsInt =0;
		string positiveNumber = Console.ReadLine();
		
		positiveNumberAsInt = int.Parse(positiveNumber);
		
		if(positiveNumberAsInt > 0){
			//Logic
			
			//a lot of work
			
			//a lot of lines
			
			//makes this very long
		}
		else{
			Console.WriteLine("Bad input");
		}
		
		
		
		//Doesn't this way look better?
		if(positiveNumberAsInt<0){
			Console.WriteLine("Bad Input");
			return;
		}
		
			//Logic
			
			//a lot of work
			
			//a lot of lines
			
			//makes this very long
	} 
}
      

File: ch4 supplemental/repl6.js
var posInt = prompt("Please enter a positive number");

if(posInt >0){
	//we have logic that runs here
	
	//it would be a lot of logic
	
	//more logic
	
	//lots of Math
	console.log("thanks for inputing a positive number")
}
else{
	console.log("Bad input");
	return;
}



////what about this way instead?

var posInt = prompt("Please enter a positive number");

if(posInt<0){
	console.log("Bad input");
	return;
}

//then our logic runs
//all that logic from above
//many lines of math and logic
console.log("thanks for inputing a positive number")